about/list_of_features.rst

```diff
@@ -0,0 +1,510 @@
+- Kinematic bodies.
+- Joints.
+- Areas to detect bodies entering or leaving it.
+
+**Collision detection:**
+
+- Built-in shapes: line, box, circle, capsule.
+- Collision polygons (can be drawn manually or generated from a sprite in the editor).
+
+3D graphics
+^^^^^^^^^^^
+
+Vulkan renderer.
+
+- HDR rendering with sRGB.
+- Perspective, orthographic and frustum-offset cameras.
+
+**Physically-based rendering:**
+
+- Follows the Disney PBR model.
+- Uses a roughness-metallic workflow with support for ORM textures.
+- Normal mapping.
+- Parallax/relief mapping with automatic level of detail based on distance.
+- Sub-surface scattering and transmittance.
+- Proximity fade (soft particles) and distance fade.
+- Distance fade can use alpha blending or dithering to avoid going through
+  the transparent pipeline.
+- Dithering can be determined on a per-pixel or per-object basis.
+
+**Real-time lighting:**
+
+- Directional lights (sun/moon). Up to 4 per scene.
+- Omnidirectional lights.
+- Spot lights with adjustable cone angle and attenuation.
+- Adjustable light "size" for fake area lights (will also make shadows blurrier).
+- Lights are rendered with clustered forward optimizations to decrease their
+  individual cost.
+
+**Shadow mapping:**
+
+- *DirectionalLight:* Orthogonal (fastest), PSSM 2-split and 4-split.
+  Supports blending between splits.
+- *OmniLight:* Dual parabolid (fast) or cubemap (slower but more accurate).
+  Supports colored projector textures in the form of panoramas.
+- *SpotLight:* Single texture. Supports colored projector textures.
+- Shadow pancaking to decrease the amount of visible shadow acne and peter-panning.
+- PCSS-like shadow blur based on the light size and distance from the surface
+  the shadow is casted on.
+- Adjustable blur on a per-light basis.
+
+**Global illumination with indirect lighting:**
+
+- Baked lightmaps (fast, but can't be updated at run-time).
+   - Lightmaps are baked on the GPU using compute shaders.
+- GI probes (slower, fully real-time). Supports reflections.
+- Signed distance field GI (intermediate, supports dynamic lights but not
+  dynamic occluders). Supports reflections.
+- Global illumination uses a deferred pass to allow for adaptive subsampling.
+
+**Reflections:**
+
+- Fast baked reflections or slow real-time reflections using ReflectionProbe.
+  Parallax correction can optionally be enabled.
+- Reflection techniques can be mixed together for greater accuracy.
+
+**Sky:**
+
+- Panorama sky (using an HDRI).
+- Procedural sky (with "high quality" or "real time" options for faster rendering).
+- Physically-based sky (with "high quality" or "real time" options for faster rendering).
+   - When using a physically-based sky, it also adjusts the color automatically
+     depending on the sun's angle.
+- In ProceduralSky and PhysicalSky, sun direction is automatically adjusted
+  based on the first DirectionalLight's rotation.
+- Support for custom sky shaders.
+
+**Fog:**
```

Comment:
Fog is going to change a lot over the next couple weeks. reduz still hasnt merged the fog [PR](https://github.com/godotengine/godot/pull/36989)

Reply:
I described what the fog PR you linked implements :slightly_smiling_face: 

Hopefully, this will remain correct once the PR is merged.

Reply:
There will be more to add. Reduz' volumetric work will likely fit in here. Plus once he is done with that I would like to add a proper atmospheric model that can be used for "fog" and for sky.